.github/workflows/pom_update.yml

```diff
@@ -26,12 +26,20 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         git checkout -b $BRANCH
+        # Get the current version
+        CURR_VERS=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's/-SNAPSHOT$//')
+        echo $CURR_VERS
+        # Increment the version
         ./mvnw build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion}-SNAPSHOT
         git add pom.xml
-        VERS=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout|tr -d SNAPSHOT|tr -d "-")
-        # Replace version in README.md look for >, :, ` (greater than, colon, backtick) followed by 2 period number period number and replace it with the leading char and new version
-        sed -i -E 's/([\>`\:]{1})2\.[0-9]\.[0-9]/\1'"$VERS"'/g' README.md
+        # Get the new version
+        VERS=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's/-SNAPSHOT$//')
+        echo $VERS
+        # Replace version in README.md look for >, : (greater than, colon) followed by "2" period number period number (ex: 2.4.2) and replace it with the same leading char and current version
+        sed -i -E 's/([\>\:]{1})2\.[0-9]\.[0-9]/\1'"$CURR_VERS"'/g' README.md
```

Comment:
It's hard for me to understand what this line should do.
But it fails on my Mac:

```
> sed -i -E 's/([\>\:]{1})2\.[0-9]\.[0-9]/\1'"$CURR_VERS"'/g' README.md
sed: 1: "s/([\>\:]{1})2\.[0-9]\. ...": \1 not defined in the RE
```

And README doesn't contains combination `>:`.

Reply:
sed behaves differently on Mac.

I ran into that the other day too. You need to `brew install gnu-sed` then use `gsed`.